doc/nvim-tree-lua.txt

```diff
@@ -131,6 +131,35 @@ Values may be functions. Warning: this may result in unexpected behaviour.
           git_placement = "before",
           padding = " ",
           symlink_arrow = " ➛ ",
+          show = {
+            file = true,
+            folder = true,
+            folder_arrow = true,
+            git = true,
+          },
+          symbols = {
```

Comment:
`renderer.icons.symbols` was the best name I could think of. Ideas welcome.

Reply:
sounds good to me :)

Reply:
Sorry... changed my mind... `glyphs` now.

Matches https://www.nerdfonts.com/ wording.